README.md

```diff
@@ -1,10 +1,75 @@
 # HTTP reply on Unikraft
 
-To build and run this application please use the `kraft` script:
+This application starts an HTTP server.
 
-    pip3 install git+https://github.com/unikraft/kraft.git
-    mkdir my-first-unikernel && cd my-first-unikernel
-    kraft up -p PLATFORM -m ARCHITECTURE httpreply
+To configure, build and run the application you need to have [kraft](https://github.com/unikraft/kraft) installed.
 
-For more information about `kraft` type ```kraft -h``` or read the
-[documentation](http://docs.unikraft.org).
+To be able to interact with the HTTP server, configure the application to run on the KVM platform:
+```
+$ kraft configure -p kvm -m x86_64
+```
+
+Build the application:
+```
+$ kraft build
+```
+
+And, finally, run the application:
+```
+$ kraft build
+[...]
+Listening on port 8123...
+```
+
+The server listens on port `8123` but we can't access it, as the virtual machine doesn't have a (virtual) network connection to the host system and it doesn't have an IP address.
+So we have to create a connection and assign an IP address.
```

Comment:
This text should be put towards the end of the readme, it's a bit duplicate.

Reply:
My goal was to show that if just run the application it won't provide you the HTTP server functionality: there's not network connection and no IP address. This is the reasoning behind using a virtual network and virtual bridge: to get a connection to the HTTP server application running inside a Unikraft VM.

Reply:
Yes, but running `kraft build` won't start it running though?

Reply:
@nderjung , solved.